### PR TITLE
New Recipe Sourmash

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/sourmash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - sourmash = sourmash_lib.__main__:main
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - screed >=0.9
+    - pyyaml >=3.11
+    - libgcc
```
Comment (bgruening) [inline recipes/sourmash/meta.yaml]: libgcc is not needed as build-time dependencies, isn't it?

Comment (bgruening) [inline recipes/sourmash/meta.yaml]: Do you need gcc here?

Comment (Takadonet) [inline recipes/sourmash/meta.yaml]: Was getting run time  error 
"SERR ImportError: libstdc++.so.6: cannot open shared object file: No such file or directory"

Comment (bgruening) [inline recipes/sourmash/meta.yaml]: In build? Are you sure? In runtime, this can be but build-time deps?

Comment (Takadonet) [inline recipes/sourmash/meta.yaml]: Here is the failed build before adding libgcc . 

https://travis-ci.org/phac-nml/bioconda-recipes/jobs/223166549#L1859

Error near the bottom.

Comment (bgruening) [inline recipes/sourmash/meta.yaml]: Yes, this is only for run-time deps not for build-time.

Comment (Takadonet) [inline recipes/sourmash/meta.yaml]: removed and test pass.

**Comment by bgruening** [comment]
@Takadonet thanks!

**Pull request merged**